dev/tasks/tasks.yml

```diff
@@ -1494,10 +1494,11 @@ tasks:
 
 {% for python_version, pandas_version, numpy_version, cache_leaf in [("3.8", "1.0", "1.19", True),
                                                                      ("3.9", "latest", "latest", False),
+                                                                     ("3.10", "latest", "<2.0.0", False),
```

Comment:
Alternatively, we could also use `1.*` here, which will still work with the `pip install numpy==${numpy}`?

Reply:
I did try `1.*` first and it broke github actions / crossbow triggers. I think the `*` is a reserved char somewhere. I could find a different way to parameterize the numpy job name to solve it though, if you think that is a better solution?

Reply:
https://github.com/apache/arrow/pull/42189#issuecomment-2174403654